src/EnergyPlus/WindowManager.cc

```diff
@@ -7782,6 +7792,9 @@ namespace WindowManager {
                 CalcNominalWindowCond(state, ThisNum, 1, NominalConductanceWinter, SHGCWinter, TransSolNorm, TransVisNorm, errFlag);
                 if (errFlag == 1 || errFlag == 2) continue;
                 state.dataHeatBal->NominalU(ThisNum) = NominalConductanceWinter;
+                // Need to have this because of window assembly reports (Simon)
+                state.dataConstruction->Construct(ThisNum).SummerSHGC = SHGCSummer;
```

Comment:
`SHGCSummer` hasn't been calculated inside this loop. It will always be zero here. Should there be another call to `CalcNominalWindowCond` here for summer? 
Maybe if this actually set `SHGCSummer` to a non-zero value here, then the checks in HeatBalanceSurfaceManager line 745ff might not be needed?
And not necessarily in this PR, but the 1 or 2 arguments passed to `CalcNominalWindowCond` for summer/winter scream for an enum.